packages/server/src/admin/admin-entities.ts

```diff
@@ -0,0 +1,19 @@
+import { AdminEntity } from 'nestjs-admin';
+import { CourseModel } from 'course/course.entity';
+import { QueueModel } from 'queue/queue.entity';
+import { UserModel } from 'profile/user.entity';
+
+export class CourseAdmin extends AdminEntity {
+  entity = CourseModel;
+  listDisplay = ['id', 'name'];
+}
+
+export class QueueAdmin extends AdminEntity {
+  entity = QueueModel;
+  listDisplay = ['id', 'room', 'courseId'];
+}
+
```

Comment:
Should we add the OfficeHours entity here? Just incase they get switched up last minute on the ical and we haven't re-parsed

Reply:
going to find some way to add a button to trigger reparse, but later ticket